educhain/utils/audio_utils.py

```
@@ -0,0 +1,723 @@
+            # Make API request
+            response = requests.post(url, headers=headers, json=payload, timeout=120)
+            
+            # Check for errors
+            if response.status_code != 200:
+                error_msg = f"API returned status {response.status_code}"
+                try:
+                    error_data = response.json()
+                    error_msg = error_data.get('error', error_msg)
+                    if isinstance(error_msg, dict):
+                        error_msg = error_msg.get('message', str(error_msg))
+                except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)